miniapps/navier/navier_cht.cpp

```diff
@@ -53,7 +53,9 @@
 //
 // The one-sided coupling between the two domains is via transfer of the
 // advection velocity (u) from fluid domain to thermal domain at each time step.
-// mpirun -np 4 navier_cht -r1 3 -r2 2 -np1 2 -np2 2
+//
+// Sample run:
+//   mpirun -np 4 navier_cht -r1 3 -r2 2 -np1 2 -np2 2
```

Comment:
minor: maybe add `> `  for consistency with the other changes
```
  >  mpirun -np 4 navier_cht -r1 3 -r2 2 -np1 2 -np2 2
```

Reply:
The `>` in other places is used to suppress the sample-run from being detected by the script `sample-runs.sh`. In this case we do not want to suppress it. We probably use different characters in different places for this purpose: `>`, `*`, and maybe others -- any non-space symbol works.